src/Security/ContaoAuthenticator.php

```diff
@@ -28,6 +30,21 @@
 class ContaoAuthenticator implements SimplePreAuthenticatorInterface
 {
     /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * Constructor.
+     *
+     * @param ContainerInterface $container The service container
+     */
+    public function __construct(ContainerInterface $container)
```

Comment:
That's wrong. It was possible to instantiate the class without passing a container object before and it is no longer now. Definitely an API change.